docs/SPIDER_TYPE.md

```diff
@@ -0,0 +1,23 @@
+## Types of spiders
```

Comment:
Could we add a general comment that it is OK for multiple spiders of different types to extract duplicate features, regardless of whether different fields of data are available. For example, a brand spider may extract a feature from a hotel chain website, and the same hotel feature may be additionally extracted from a government spider for registration of hotels, and additionally extracted from a hotel aggregator service spider.

Reply:
Maybe, but it's not new to this PR and I don't see it flowing in this file.